sdk/aqueduct/api_client.py

```diff
@@ -166,6 +186,7 @@ def _test_connection_protocol(self, try_http: bool, try_https: bool) -> bool:
                 self._test_url(url)
                 return False
             except Exception as e:
+                print(e)
```

Comment:
I think we can probably just do `logger.error()` here, which already includes the exception.